doc/apps/dash.md

```diff
@@ -104,6 +109,27 @@ ploomber-cloud deploy --watch
 To ensure your app doesn't break on re-deployments, pin your [dependencies.](pin-dependencies)
 ```
 
+(customize-deployment)=
+## Customize deployment
+
+To customize the Python version or base image for deployment, you may use [Docker deployment](./docker.md).
```

Comment:
add some more info on why someone would use docker over the native option

docker is more flexible: you can install dependencies that are not available via pip, control the python version, etc

Reply:
[Fixed](https://github.com/ploomber/doc/pull/196/files#diff-06950dd861443fb8233dd171ce6a68c04fc5e21fac3af4e66e757a1f8ae808d4R108-R118)